scripts/setup.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+# ASCII art
+echo "  _           _           _             "
+echo " | |         | |         | |            "
+echo " | | ___   __| | ___  ___| |_ __ _ _ __ "
+echo " | |/ _ \ / _  |/ _ \/ __| __/ _  |  __|"
+echo " | | (_) | (_| |  __/\__ \ || (_| | |   "
+echo " |_|\___/ \__ _|\___||___/\__\__ _|_|   "
+echo ""
+
+# Declare a log file to capture detailed logs and a temp directory
+LOGFILE="$HOME/lodestar-app-temp/logfile.log"
```

Comment:
why do we need to write to a log file? simply printing the logs to stdout (terminal) seems sufficient here

Reply:
I tried to make it similar to celestiaorg, they were using it so added this, now removed this

Reply:
I haven't checked their script in detail just referenced it as the overall approach is good, I don't like how they do the installation though, too much unnecessary stuff imo, it should be as simple as possible